draft-tus-resumable-uploads-protocol.md

```diff
@@ -164,7 +164,9 @@ This procedure is designed to be compatible with a regular upload. Therefore all
 
 The client MUST use the same method throughout an entire upload. The server SHOULD reject the attempt to resume an upload with a different method with `400 (Bad Request)` response.
 
-The request MUST include the `Upload-Token` header which uniquely identifies an upload.
+The client MUST NOT perform multiple Upload Transfer Procedures in parallel.
```

Comment:
Does this mean globally or for the same upload-token? Is there a reason we want to prohibit multiple uploads of different files at the same time?

Reply:
Thanks, fixed